app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { module } from 'angular';
+import { react2angular } from 'react2angular';
+import {IEcsCapacityProviderStrategy, IEcsServerGroupCommand} from '../../serverGroupConfiguration.service';
+import { HelpField, withErrorBoundary } from  '@spinnaker/core';
+
+export interface ICapacityProviderProps {
+  command: IEcsServerGroupCommand;
+  notifyAngular: (key: string, value: any) => void;
+  configureCommand: (query: string) => PromiseLike<void>;
```

Comment:
this func doesn't seem to be used in this component?

Reply:
Actually we will need this func as a part of the new enhanced UI, so I thought we will need it anyway. But it makes sense to remove it as it is not a part of this implementation. We should be migrating things as is. I will remove that func.